X/Xorg_libXinerama/build_tarballs.jl

```diff
@@ -38,4 +38,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Comment:
Changing compat bounds ***requires*** changing the version number of the package itself (Pkg won't be able to deal with that otherwise)

Reply:
Bumped the version in 870e0f32.